app/src/main/java/com/google/samples/apps/sunflower/adapters/GardenPlantingAdapter.kt

```diff
@@ -43,10 +43,8 @@ class GardenPlantingAdapter :
     }
 
     override fun onBindViewHolder(holder: ViewHolder, position: Int) {
-        getItem(position).let { plantings ->
-            with(holder) {
-                bind(plantings)
-            }
+        with(holder) {
+            bind(getItem(position))
```

Comment:
This is good, but maybe we should also remove the `with` here, since there's only one line inside it. That makes this function really straight forward:

```
    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
        holder.bind(getItem(position))
    }
```

Reply:
great! thx!